modules/programs/btop.nix

```diff
@@ -0,0 +1,56 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.btop;
+
+  formatLine = n: v:
+    let
+      formatValue = v:
+        if builtins.isBool v then
+          (if v then "True" else "False")
+        else if builtins.isString v then
+          ''"${v}"''
+        else
+          toString v;
+    in "${n} = ${formatValue v}";
```

Comment:
```suggestion
  finalConfig = ''
    ${generators.toKeyValue { } cfg.settings}
    ${optionalString (cfg.extraConfig != "") cfg.extraConfig}
  '';
```

Reply:
Yeah, if possible then this would be ideal. Though, I do notice that the default configuration of `toKeyValue` serializes boolean values as `true` and `false` (i.e., lowercased) and strings are serialized without the surrounding quotes. So it may be necessary to override the `mkKeyValue` or `mkValueString` functions.

Reply:
Indeed I need this specific behavior for boolean and string values.
How could I overwrite the `mkKeyValue` and `mkValueString` functions ?

Reply:
I gave it a shot, tell me if it is good for you

Reply:
Looks good 👍